Sample_v3/Samples/SimpleChat/SimpleChatViewController.swift

```diff
@@ -72,14 +71,8 @@ final class SimpleChatViewController: UITableViewController, ChannelControllerDe
     ///
     /// The method below receives a `TypingEvent` and updates the view controller's `navigationItem.prompt` to show that an user is currently typing.
     ///
-    func channelController(_ channelController: ChannelController, didReceiveTypingEvent event: TypingEvent) {
-        guard let user = channelController.dataStore.user(id: event.userId) else { return }
-        
-        if event.isTyping {
-            navigationItem.prompt = "\(user.name ?? event.userId) is typing..."
-        } else {
-            navigationItem.prompt = ""
-        }
+    func channelController(_ channelController: ChannelController, didReceiveTypingEvent: TypingEvent) {
```

Comment:
If we come up with the way of delivering typing updates via `didUpdateChannel` (in the same manner it is currently done for `ChannelListController`) we can completely get rid of `didReceiveTypingEvent`